src/pages/MembersPage/getMembersPageData.ts

```diff
@@ -0,0 +1,110 @@
+import { order_by } from 'lib/gql/__generated__/zeus';
+import { client } from 'lib/gql/client';
+
+import { Awaited } from 'types/shim';
+
+export const getMembersPageData = async (circleId: number) => {
```

Comment:
maybe we should rename this to `CircleMembers` or `OrgMembers` now to differentiate these two...

Reply:
I'll do this renaming in a later PR